static/js/components/CommentVoteForm_test.js

```diff
@@ -91,23 +90,34 @@ describe("CommentVoteForm", () => {
     )
   }
 
-  it("should have tooltips, if the user is anonymous", () => {
+  it("should have a LoginPopup, if the user is anonymous", () => {
     sandbox.stub(utilFuncs, "userIsAnonymous").returns(true)
     const wrapper = renderForm()
-    wrapper.find(ReactTooltip).forEach(tooltip => {
-      assert.equal(tooltip.text(), votingTooltipText)
-    })
+    wrapper.find(LoginPopup).exists()
   })
 
-  it("should not have any tooltips, if the user is not anonymous", () => {
+  it("should not have a LoginPopup, if the user is not anonymous", () => {
     sandbox.stub(utilFuncs, "userIsAnonymous").returns(false)
     assert.isNotOk(
       renderForm()
-        .find(ReactTooltip)
+        .find(LoginPopup)
         .exists()
     )
   })
 
+  it("clicking a vote button should set visible state to true and not trigger vote function", () => {
+    sandbox.stub(utilFuncs, "userIsAnonymous").returns(true)
+    const wrapper = renderForm()
+    const voteStub = upvoteStub
+    wrapper.find(".upvote-button").simulate("click")
+    sinon.assert.notCalled(voteStub)
+    assert.deepEqual(wrapper.state(), {
+      downvoting:   false,
+      upvoting:     false,
+      popupVisible: true
+    })
```

Comment:
maybe just add a quick assert that the `LoginPopup` has the `visible` prop when we have this state?